scripts/generate-iscsi-iqn

```diff
@@ -55,7 +58,10 @@ xe host-param-set uuid=${INSTALLATION_UUID} iscsi_iqn="${IQN}"
 echo Set iSCSI IQN: "${IQN}"
 echo IQN=\'"${IQN}"\' >${IQN_CONF}
 
-# Ensure changes are synced to disk
-xe pool-sync-database
+
+if [ ! "$UPGRADE" = "true" ]; then
```

Comment:
Would it be clearer to use `[  "$UPGRADE" != "true" ]`?

Reply:
Given that `UPGRADE` is set to `false` unconditionally above, does this need a test at all?

Reply:
yes, we need the check - `UPGRADE` potentially gets sourced by this:

```
[ -r ${FIRSTBOOT_DATA_DIR}/host.conf ] && . ${FIRSTBOOT_DATA_DIR}/host.conf
```